Stream/Stream.sln

```diff
@@ -44,6 +37,27 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PartitionedProducer", "Part
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PartitionedConsumer", "PartitionedConsumer\PartitionedConsumer.csproj", "{E78ED292-CD60-4245-BD86-5EC956659895}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "UsageCost", "UsageCost", "{313BB13F-52B0-4342-8E30-497E340913B5}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "UsageSender", "UsageCost\UsageSender\UsageSender.csproj", "{89A079BD-937F-49DE-B7AA-920388598010}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "UsageProcessor", "UsageCost\UsageProcessor\UsageProcessor.csproj", "{C266230F-E647-477C-B4E0-86F0836F174E}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "UsageLogger", "UsageCost\UsageLogger\UsageLogger.csproj", "{60209297-C680-453A-9DCA-65BDFB8D3E51}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "kubernetes", "kubernetes", "{A0AD1847-A989-4D3D-9F69-228156FAD3BB}"
+	ProjectSection(SolutionItems) = preProject
+		UsageCost\usage-cost-stream.yaml = UsageCost\usage-cost-stream.yaml
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CloudDataflow", "CloudDataflow", "{9951BCFB-4AD5-4B62-9D83-F40D4D53ED7A}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CloudDataflowK8s", "CloudDataflowK8s", "{2DDC16E3-3F84-4D81-A3D1-A85377E116CD}"
```

Comment:
It looks like you removed the `CloudFoundry` and `K8s` solution folders, but left K8s in the project names, was all of that intentional? I don't know if it matters, just asking

Reply:
I replaced them with CloudDataFlowCF(just now) and CloudDataFlowK8s. Initially I was going to have a separate folder for K8s and CF but the UsageCost Standalone sample works as is for all environments - local, K8s and CF. Hence the name change.